build.ps1

```diff
@@ -355,6 +355,21 @@ try
         $env:AZURE_IOT_LOCALPACKAGES = $packageTempPath
     }
 
+    if ($localPackagesAvailableForTesting)
```

Comment:
Moving this up the confirm the local packages directory immediately after the solution has been built.